src/test/scala/symsim/examples/concrete/mountaincar/MountainCarSpec.scala

```diff
@@ -0,0 +1,65 @@
+package symsim
+package examples.concrete.mountaincar
+
+import symsim.concrete.Randomized.given
+import CanTestIn.given
+import org.scalatest.*
+import prop.*
+import org.scalacheck.Arbitrary.*
+import org.scalacheck.Gen
+import org.scalatest.prop.Whenever
+import org.scalatest.*
+import org.scalacheck.Prop.{exists, forAll, forAllNoShrink, propBoolean}
+import examples.concrete.mountaincar.CarState
+import examples.concrete.mountaincar.MountainCar
+import Math.cos
+
+/** Sanity tests for Randomized as a Scheduler */
+class MountainCarSpec
+  extends org.scalatest.freespec.AnyFreeSpec,
+    org.scalatestplus.scalacheck.Checkers:
+
+  given PropertyCheckConfiguration =
+    PropertyCheckConfiguration(minSuccessful = 100)
+
+  "Sanity checks for symsim.concrete.breaking" - {
+
+    // Generators of test data
+    val positions = Gen.choose[Double] (-1.2, 0.5)
+    val velocities = Gen.choose[Double] (-1.5,1.5)
+    val negPos = Gen.choose[Double] (-1.2, -1.1)
+    val actions = Gen.oneOf (MountainCar.instances.enumAction.membersAscending)
+
+
+    // Tests
+
+    "Achieving goal state from state (0.0, 0.0) is impossible" in check {
```

Comment:
"Achieving the goal state from state (0.0, 0.0) with a single action is impossible"